tests/conftest.py

```diff
@@ -30,7 +30,10 @@ def example_cases(tmp_path_factory):
     Returns the path to that copy.
```

Comment:
I can't put the comment above, but is there a reason that you giving `scope="session"` here?  The default, or explicitly, `scope="function"` would seem to me to be better, so we get everything fresh for each test invocation as we run through the parametrize.  Otherwise all the copies we do will build on each other and the behavior of the test could depend on what did or didn't run before.